### DISCUSSION TITLE
Is their a way to go to a specific time stamp in an mp4 wallpaper through lively command line?

### DISCUSSION THREAD

**Discussion by Ucef27** [rocksdanister/lively#383] [Q&A] [answered]
Ultimately, I would like to skip to specific to time stamps using auto hotkey.

**Comment by rocksdanister** [comment, marked as answer]
Good idea.. next update will have this feature: https://github.com/rocksdanister/lively/commit/e166ba4521863b9a9747cf9392ac72bac5b09b97
First version will only take percentage value(0-100) for seeking.

**Reply by Ucef27** [reply]
Ok, thanks

**Comment by rocksdanister** [comment]
Beta build with this feature is out:
https://github.com/rocksdanister/lively-beta/releases
